game/src/backend/popup_messages.rs

```diff
@@ -1,10 +1,11 @@
 /// This is the text for the popups that appears in the top left corner.
 
-pub const WARNINGS: [&str; 4] = [
+pub const WARNINGS: [&str; 5] = [
     "$m : Ein Meteorit ist auf dem Weg!",
     "$m : Der Strom ist ausgefallen!",
     "$m : Ein Sandsturm ist auf dem Weg!",
     "Machine $m ist ausgefallen!",
+    "Lebensregeneration gestartet",
```

Comment:
Ich dachte eigentlich es würden in den Warnings random Sprüche sein, wo man bei jeder Maschine sagen kann send_popup(random(WARNINGS)) und dann $m ersetzt wird...

Reply:
abgeändert